### DISCUSSION TITLE
Motor control

### DISCUSSION THREAD

**Discussion by Almenau14** [Severson-Group/AMDC-Hardware#243] [General]
Hi,

I am an undergraduate student at the State University of Santa Catarina (UDESC) and currently a research assistant under Professor Dr. Ademir Nied. I am working on a project to develop an induction motor drive using AMDC. In the material available on GitHub, we found the uInverter, which emulates an RL load, and we plan to use it to extract data with AMDC and gain familiarity with the system.

I would like to inquire if you have any pre-developed boards suitable for driving motors up to 1 kW. If such a board is available, could you please provide information on its specifications and availability? Having this board would be extremely beneficial for the progress of our project.

Thank you in advance for your assistance. I look forward to your guidance and additional information.

Best regards,

[Vinícius Cidral de Almenau]
Undergraduate Student – Electrical Engineering
State University of Santa Catarina (UDESC)
Research Assistant to Professor Ademir Nied

**Comment by codecubepi** [comment]
Hi Vinícius! We're glad to hear about your ongoing interest in the AMDC platform. My name is Patrick; I've been contributing to the AMDC ecosystem for two years as an undergrad at UW-Madison, and I am now starting my master's while continuing to use the platform.

The AMDC + uInverter is a great way to become familiar with the system. Hopefully you have found our documentation website, [docs.amdc.dev](https://docs.amdc.dev/), which has some onboarding tutorials for the whole AMDC ecosystem. Information on the uInverter board can be found [here](https://docs.amdc.dev/accessories/uinverter/index.html), which pairs nicely with the Voltage Source Inverter tutorial [here](https://docs.amdc.dev/getting-started/tutorials/vsi/index.html).

I don't believe the Severson Group has any publicly-available board designs for an inverter/drive for a real motor. This is because many of the group's research projects use drives that are specialized to the needs of that individual project.

**Comment by codecubepi** [comment]
Hi Vinícius,

I'd also like to follow up and let you know that we've just released some new firmware for the AMDC platform. From now own, we'll be announcing new firmware releases in the GitHub Discussions tab in our AMDC-Firmware repository. You can find the announcement for the latest release [here](https://github.com/Severson-Group/AMDC-Firmware/discussions/418).

Have you received the hardware you ordered yet?

**Comment by Almenau14** [comment]
Good morning,

We are currently facing budget issues, but we will likely receive research
funding next month and will resume the manufacturing process with PCBWAY.
Because of this, I am dedicating my efforts to understanding the firmware
and plan to test the new configuration soon.

Em qua., 18 de set. de 2024 às 14:28, Patrick Nowakowski <
***@***.***> escreveu:

**Comment by Almenau14** [comment]
Good night!

Hi Patrick, we are excited! Today we completed the payment and are just
waiting for PCBWay to send us the AMDC + uInverter PCB. Now I will start
studying the firmware to implement motor control.

Em qui., 19 de set. de 2024 às 08:40, Vinicius Almenau <
***@***.***> escreveu:

**Reply by codecubepi** [reply]
That's great! Glad to hear a positive update.
